Servers/UI/OJS.Servers.Ui/ClientApp/src/components/administration/tests/test-form/TestForm.tsx

```diff
@@ -57,6 +59,14 @@ const TestForm = (props: ITestFormProps) => {
 
     useDisableMouseWheelOnNumberInputs();
 
+    useEffect(() => {
+        if (isSuccessfullyCreated && onSuccess) {
+            setTimeout(() => {
+                onSuccess();
+            }, 500);
```

Comment:
It would be very helpful to add a comment explaining why timeout is used here.

Reply:
Done.